contracts/TopShot.cdc

```diff
@@ -380,6 +380,51 @@ pub contract TopShot: NonFungibleToken {
             return <-newCollection
         }
 
+        pub fun mintMomentWithSubEdition(playID: UInt32, subEditionID: UInt32): @NFT {
+            pre {
+                self.retired[playID] != nil: "Cannot mint the moment: This play doesn't exist."
+                !self.retired[playID]!: "Cannot mint the moment from this play: This play has been retired."
+            }
+
+            // Gets the number of Moments that have been minted for this Play
+            // to use as this Moment's serial number
+            let subEditionCap = TopShot.account.getCapability<&{AdminSubEdition}>(/private/AdminSubEdition)
```

Comment:
In other places I've seen this in a single line:
`let subEdition = TopShot.account.getCapability<&{AdminSubEdition}>(/private/AdminSubEdition).borrow()!`
Not sure what is considered idiomatic Cadence